pkg/operator/ceph/object/rgw.go

```diff
@@ -71,6 +71,12 @@ func (c *clusterConfig) createOrUpdateStore() error {
 		return errors.Wrap(err, "failed to start rgw pods")
 	}
 
+	objContext := NewContext(c.context, c.store.Name, c.store.Namespace)
```

Comment:
It looks like you're just using the context of `objContext` only so you don't actually need a new `objContext` context if `enableRGWDashboard` were a method of the `clusterConfig` object. With that you could use the context this way: `c.context`.

All of this goes for:

* checkDashboardUser
* enableRGWDashboard
* disableRGWDashboard

Reply:
> It looks like you're just using the context of `objContext` only so you don't actually need a new `objContext` context if

@leseb 
Apis `CreateUser()` and `DeleteUser()` is used for the `admin` which needs `objcontext`.

 `enableRGWDashboard` were a method of the `clusterConfig` object. With that you could use the context this way: `c.context`.
> 
> All of this goes for:
> 
> * checkDashboardUser
> * enableRGWDashboard
> * disableRGWDashboard

Reply:
Where? I don't see `CreateUser` in this file and the creation of the context is self-contained in `createOrUpdateStore`. So I don't understand your reply.

Reply:
> Where? I don't see `CreateUser` in this file and the creation of the context is self-contained in `createOrUpdateStore`. So I don't understand your reply.

@leseb in the functions `enableRGWDashboard` and `disableRGWDashboard`, I am not only consuming `clusterd.Context` from `clusterConfig` but also the `Context`(object store context)  and pass`CreateUser()` and `DeleteUser()` respectively.  In that case I need to create new object store context in the `enableRGWDashboard()` and `disableRGWDashboard()`.